### ISSUE TITLE
Issue with new player design

### ISSUE THREAD

**Issue by SgtChrischi** [YePpHa/YouTubeCenter#1672]
Hey there,

if I try to use the new transparent youtube player design (by changing the value of the cookie 'VISITOR_INFO1_LIVE' to 'Q06SngRDTGA' my player stays black:
http://abload.de/img/unbenanntp1ujj.png

With YTC disabled it works:
http://abload.de/img/unbenannt296u4k.png

Would be cool if this can be fixed somehow so I can use both, transparent player and YTC.

Thanks,
Chrischi

**Comment by xur86x** [comment]
Even better would be an option in YTC to use the transparent player.

**Comment by SgtChrischi** [comment]
Ok I used an addon to change the cookie the first time.

Now I tried without addon by typing 'cookie set VISITOR_INFO1_LIVE Q06SngRDTGA' into developer console (Shift + F2 on Firefox) and now it works. But it takes some times till the video loads with (which doesnt with YTC disabled).

**Comment by Yonezpt** [comment]
The transparent player is an experimental design which you are forcing on your side and as such there will be lots of changes during the experimental trial. Trying to fix whatever issue is occuring, every time they appear, will be a waste of time if they end up not making it the default player.

It will be better to wait until they decide wether the new skin will become default or not before asking the add-on author to invest time into something that might be worthless.

**Comment by SourSkittles** [comment]
This seems to only be an issue with firefox, using YTC and the transparent player works just fine together on chrome.

**Comment by cesc1989** [comment]
Hi @Yonezpt although @SgtChrischi forced the new design, in my **Firefox 37** it appeared all of a sudden and of course YTC didn't work.

Let's make clear I could disable the new design by changing the default player option to **Flash** and YTC worked as usual.

**Comment by SgtChrischi** [comment]
After some testing with YTC and other youtube addons for firefox I found out that it is the "prevent auto-play" options which doesnt work with the new design. Disabled the feature and now YTC works with the transparant design.

**Comment by Yonezpt** [comment]
@SgtChrischi Makes sense, the player is not complete yet and it appears that there is no framework for when the player is not allowed to autoplay. Either way, it is still under work and will change frequently. It is best to wait until they decide what they will do with it.

**Comment by SgtChrischi** [comment]
Yes, I think I'll need to wait. But I found another addon where stop autoplay works with the new design, so I use 2 addons until it works with YTC only.

**Comment by theaulddubliner** [comment]
@SgtChrischi What's the name of that add-on?

**Comment by SgtChrischi** [comment]
YouTube High Definition v37.0 (by Baris Derin)

**Comment by theaulddubliner** [comment]
Cheers - I'll check it out.
